From 2a4272291afd44ed1cbb1b4fa13c5d992bdd24cf Mon Sep 17 00:00:00 2001
From: Dmitry Kazakov <dimula73@gmail.com>
Date: Wed, 18 Aug 2021 14:06:05 +0300
Subject: [PATCH] Skip looking for Png2Ico when IcoTool is found

This extra search of the package is listed in the build summary
and makes developers confused, because the library is not actually
needed anymore.
---
 modules/ECMAddAppIcon.cmake | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/modules/ECMAddAppIcon.cmake b/modules/ECMAddAppIcon.cmake
index c552c16b..fb203c5f 100644
--- a/modules/ECMAddAppIcon.cmake
+++ b/modules/ECMAddAppIcon.cmake
@@ -170,6 +170,11 @@ function(ecm_add_app_icon appsources)
         set(saved_CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}")
         set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${ECM_FIND_MODULE_DIR})
         find_package(IcoTool)
+
+        if (NOT IcoTool_FOUND)
+            find_package(Png2Ico)
+        endif()
+
         set(CMAKE_MODULE_PATH "${saved_CMAKE_MODULE_PATH}")
 
         function(create_windows_icon_and_rc command args deps)
-- 
2.37.1.windows.1

